src/Model/User.php

```diff
@@ -1332,39 +1309,27 @@ public static function getStatistics()
 	 * @param int    $count Count of the items per page (Default is @see Pager::ITEMS_PER_PAGE)
 	 * @param string $type  The type of users, which should get (all, bocked, removed)
 	 * @param string $order Order of the user list (Default is 'contact.name')
-	 * @param string $order_direction Order direction (Default is ASC)
+	 * @param bool   $descending Order direction (Default is ascending)
```

Comment:
I fought with myself when creating this. And I came to the definition that when no "ASC" or "DESC" value is set in SQL, then it is ascending. So an `["order" => ["field"]]` would be ascending. So when you want to have it descending, some `["order" => ["field" => false]]` wouldn't look right in my eyes. (Since I assume "no value" with "null" or "false", but not with "true". So "true" had to be the opposite to "no value", which means descending.

Reply:
I have the opposite reasoning, in my mind, `["order" => ["field"]]` and `["order" => ["field" => true]]` are similar because they both affirm the positive existence of the value "field" in the array "order".

Too late to change project-wise, I suppose, thank you for the elaboration.